spec/terminology.md

```diff
@@ -3,17 +3,74 @@
+
+[[def: autonomic identifier (AID)]]
+~ A self-managing identifier that is self-certifying (i.e., self-authenticating) and cryptographically bound to a [[ref: key event log]] ([[ref: KEL]]). An AID is either non-transferable or transferable. A non-transferable AID does not support key rotation while a transferable AID support key rotation using a key [[ref: pre-rotation]] mechanism that enables the AID to persist in spite of the evolution of its key state.
+
+[[def: compact Event Streaming Representation (CESR), CESR]]
+~ A dual text-binary encoding format that has the unique property of text-binary concatenation composability. The composability property enables the round trip conversion en-masse of concatenated primitives between the text domain and binary domain while maintaining the separability of individual primitives.
+
+[[def: controller]]
+~ A controlling entity that can cryptographically prove the control authority (signing and rotation) over an [[ref: AID]], as well as make changes on the associated [[ref: KEL]]. A controller may consist of multiple controlling entities in a multi-signature scheme.
 
 [[def: DID document]]
 ~ As defined by [DID Core](https://www.w3.org/TR/did-core/#dfn-did-documents). See section [DID Documents](#did-documents).
 
+[[def: direct replay mode]]
+~ an operational mode of the [[ref: KERI]] protocol where a controller and a verifier of an [[ref: AID]] exchange the [[ref: KEL]] of the AID directly. Upon reception of the KEL, the verifier sends a key event receipt, which includes the verifier's signature, to the controller as an acknowledgment. The signed key event receipt attests that the verifier received and verified the key event message. 
+
+[[def: inception event]]
+~ A key establishment event that provides the incepting information needed to derive an [[ref: AID]] and establish its initial [[ref: key state]].
+
+[[def: indirect replay mode]]
+~ An operational mode of the [[ref: KERI]] protocol where the [[ref: KEL]] of an [[ref: AID]] is promulgated to a verifier using witnesses that are disignated by the AID's controller. 
+
+[[def: interaction event]]
+~ A key non-establishment event that anchors external data to the [[ref: key state]] of an [[ref: AID]] as established by the most recent prior establishment event.
+
+[[def: key event]]
+~ A serialized data structure of an entry in the [[ref: key event log]]([[ref: KEL]]) for an [[ref: AID]]. A key event is either establishment or non-establshment. An establishment event changes the [[ref: key state]] of an AID while a non-establishment event does not.
```

Comment:
I think this should written from the viewpoint of the impact on the DID of the event.  However, even saying that, this description should (I think) be talking about the inception event, the rotation event, and multi-sig events.

From this, I don’t understand what “establishment” and “non-establishment” events are, nor how they impact the DID/DIDDoc.

Reply:
I will remove "establishment" and "non-establishment" from the terminology as well as simplify the descriptions for all three types of key events.